packages/itwinui-css/src/searchbox/searchbox.scss

```
@@ -12,10 +12,15 @@
   }
 }
 
-// noop https://github.com/iTwin/iTwinUI/pull/1881#discussion_r1506166537
-/* stylelint-disable-next-line block-no-empty */
 .iui-search-icon {
+  --_iui-button-icon-fill: var(--iui-color-icon-muted);
+
+  [data-iui-size='small'] & svg {
+    inline-size: 0.75rem;
```

Should this be `--iui-size-s` instead?

```suggestion
    inline-size: var(--iui-size-s);
```